charts/frontend/values.schema.json

```diff
@@ -203,6 +203,13 @@
           "exposedRoute": { "type": "string" },
           "replicas": { "type": "integer" },
           "mounts": { "type": "array" },
+          "lifecycle": {
+            "type": "object",
+            "additionalProperties": false,
+            "properties": {
+              "preStop": { "type": "object" }
```

Comment:
Also postStart hook could be allowed, see info at https://kubernetes.io/docs/concepts/containers/container-lifecycle-hooks/

Reply:
Good idea, added. 

The "best" test of it for the hello container I could come up with right now as an echo command. It won't delay the CI/CD run but it will output something showing that it's been called.